sdk/framework/openapi.go

```diff
@@ -918,8 +918,8 @@ func convertType(t FieldType) schemaType {
 		ret.baseType = "integer"
 		ret.format = "int64"
 	case TypeDurationSecond, TypeSignedDurationSecond:
-		ret.baseType = "integer"
-		ret.format = "seconds"
+		ret.baseType = "string"
```

Comment:
don't these endpoints accept both int OR string?

if that's the case, wouldn't it break those passing in ints?

Reply:
I was mostly looking at how these fields are returned in responses today. Do you have an example of where this is used as int or string in a request?

Reply:
there's at least one, still looking for a request, but this is a response that returns int or string
[framework.Path](https://github.com/hashicorp/vault/blob/b86f4759274ff9733788f42c0a393de2e867ae19/vault/logical_system_paths.go#LL1762C29-L1762C29)
[callback fcn](https://github.com/hashicorp/vault/blob/b86f4759274ff9733788f42c0a393de2e867ae19/vault/logical_system.go#L3430-L3434)

Reply:
couldn't find any requests that handle both int or string, would need to take a closer look

Reply:
The `sys/rotate/config` [example](https://github.com/hashicorp/vault/blob/b86f4759274ff9733788f42c0a393de2e867ae19/vault/logical_system.go#L3430-L3434) you mentioned seems to be a bug where two different types are returned. Unfortunately, it would not be easy to fix, since it would break backwards compatibility.

As far as requests go, I think we should be fine as long as both string and int are accepted. As long as the user provides a string, it should work.